app/Http/Controllers/Twill/BaseApiController.php

```diff
@@ -80,37 +122,231 @@ protected function getBrowserTableData(\Illuminate\Support\Collection|\Illuminat
+
+        if ($this->getIndexOption('includeScheduledInList') && $this->repository->isFillable('publish_start_date')) {
+            $columns->add(
+                ScheduledStatus::make()
+                    ->title(twillTrans('twill::lang.publisher.scheduled'))
+                    ->optional()
+            );
+        }
+
+        if ($this->moduleHas('translations') && count(getLocales()) > 1) {
+            $columns->add(
+                Languages::make()
+                    ->title(twillTrans('twill::lang.listing.languages'))
+                    ->optional()
+            );
+        }
+
+        return $columns;
+    }
+
+    protected function getBrowserTableColumns(): TableColumns
+    {
+        $columns = TableColumns::make();
+
+        if ($this->moduleHas('medias')) {
+            $columns->add(
+                ApiImage::make()
+                    ->field('thumbnail')
+                    ->rounded()
+                    ->title(twillTrans('Image'))
+            );
+        }
+
+        $columns = $columns->merge($this->additionalBrowserTableColumns());
+
+        return $columns;
+    }
+
+    public function getSideFieldSets(TwillModelContract $model): Form
+    {
+        return parent::getSideFieldSets($model)
+        // For some reason, the side form will not render unless there is a
+        // field in the default Content fieldset. 🤷
+        ->add(
+            Input::make()
+                ->name('id')
+                ->disabled()
+                ->note('readonly')
+        )
+        ->addFieldset(
+            Fieldset::make()
+                ->id('datahub')
+                ->title('Datahub')
+                ->closed()
+                ->fields([
+                    Input::make()
+                        ->name('datahub_id')
+                        ->disabled()
+                        ->note('readonly'),
+                    Input::make()
+                        ->name('source_updated_at')
+                        ->disabled()
+                        ->note('readonly'),
+                ])
+        );
+    }
+
+    protected function transformIndexItems(Collection|LengthAwarePaginator $items): Collection|LengthAwarePaginator
+    {
         if ($this->hasAugmentedModel) {
             $ids = $items->pluck('id')->toArray();
             $this->localElements = $this->repository->whereIn('datahub_id', $ids)->get();
             $items->setCollection($items->getCollection()->map(function ($item) {
                 if ($element = collect($this->localElements)->where('datahub_id', $item->id)->first()) {
                     $item->setAugmentedModel($element);
                 }
-
                 return $item;
             }));
         }
-
         return $items;
     }
 
     /**
-     * Disable sorting by default for API listings. This has to be implemented individually on each controller
+     * Disable default Twill order by `created_at`.
```

Comment:
By default, Twill tries to order index table items by `created_at`, but since this field doesn't exist on our api models, it causes issues.